src/runtime/recipe/handle.ts

```diff
@@ -201,7 +201,7 @@ export class Handle {
     return false;
   }
 
-  isResolved(options = undefined) {
+  isResolved(options = undefined): boolean {
```

Comment:
should this have the type `ToStringOptions | undefined`?

Reply:
likely yes.